read-fonts/src/tests/test_fonts/README.md

```diff
@@ -0,0 +1,6 @@
+The ttx files are the masters. To update the binaries:
+
+```shell
+# in the test_fonts dir
+for f in $(ls *.ttx); do ttx -o ${f/.ttx/.ttf} $f; done
```

Comment:
Maybe we should just include a `rebuild.sh` script? I might also keep the ttx & ttf in separate folders, but that's just me.

Reply:
Moved to separate folders. As rebuild is a one-liner I did not add an sh. I could be convinced to do so.